CORE_DEV_GUIDE.md

```diff
@@ -162,11 +162,16 @@ they should have: been part of the project for at least two months, contributed
 significant changes of their own, contributed to the discussion and
 review of others' work, and collaborated in a way befitting our
 community values. After nomination voting will happen on a private mailing list.
-While it is expected that most votes will be unanimous, a two-thirds majority of
+While it is expected that most votes will be unanimous, a majority of
 the cast votes is enough.
 
-Core developers can choose to become emeritus core developers and suspend
-their approval and voting rights until they become active again.
+## Offboarding Core Members
+
+Core developers are expected to make regular contributions to the project. Core developers
+can choose to become emeritus core developers and suspend their approval and voting rights
+until they become active again. If a core developer becomes inactive in the project for a
```

Comment:
Can we define "inactive"? Which of the following are considered (and with ANDing or ORing?):
 * commits
 * issue/PR comments
 * vote / roll call responses

Reply:
I've added a section to this point.

Reply:
Thanks.